lib/platform/bitbucket/index.ts

```diff
@@ -749,28 +749,65 @@ export async function updatePr({
       }
     );
   } catch (err) {
+    const inactiveReviewersMessage = 'reviewers: Malformed reviewers list';
+    const notMemberReviewersMessage =
+      'is not a member of this workspace and cannot be added to this pull request';
+
     if (
       err.statusCode === 400 &&
-      err.body?.error?.message.includes('reviewers: Malformed reviewers list')
+      [inactiveReviewersMessage, notMemberReviewersMessage].some((m) =>
+        err.body?.error?.message.includes(m)
+      )
     ) {
-      logger.warn(
-        { err },
-        'PR contains inactive reviewer accounts.  Will try setting only active reviewers'
-      );
+      const sanitizedReviewers: PrReviewer[] = [];
 
       // Bitbucket returns a 400 if any of the PR reviewer accounts are now inactive (ie: disabled/suspended)
-      const activeReviewers: PrReviewer[] = [];
-
-      // Validate that each previous PR reviewer account is still active
-      for (const reviewer of pr.reviewers) {
-        const reviewerUser = (
-          await bitbucketHttp.getJson<UserResponse>(
-            `/2.0/users/${reviewer.account_id}`
-          )
-        ).body;
-
-        if (reviewerUser.account_status === 'active') {
-          activeReviewers.push(reviewer);
+      if (err.body?.error?.message.includes(inactiveReviewersMessage)) {
+        logger.warn(
+          { err },
+          'PR contains inactive reviewer accounts.  Will try setting only active reviewers'
+        );
+
+        // Validate that each previous PR reviewer account is still active
+        for (const reviewer of pr.reviewers) {
+          const reviewerUser = (
+            await bitbucketHttp.getJson<UserResponse>(
+              `/2.0/users/${reviewer.account_id}`
+            )
+          ).body;
+
+          if (reviewerUser.account_status === 'active') {
+            sanitizedReviewers.push(reviewer);
+          }
+        }
+      }
+
+      // Bitbucket returns a 400 if any of the PR reviewer accounts are no longer members of this workspace
+      if (err.body?.error?.message.includes(notMemberReviewersMessage)) {
+        logger.warn(
+          { err },
+          'PR contains reviewer accounts which are no longer member of this workspace.  Will try setting only member reviewers'
+        );
+
+        // Validate that each previous PR reviewer account is still a member of this workspace
+        for (const reviewer of pr.reviewers) {
+          try {
+            const membership = await bitbucketHttp.head(
+              `/2.0/workspaces/${config.repository.split('/')[0]}/members/${
+                reviewer.account_id
+              }`
+            );
```

Comment:
Add `throwHttpErrors:false` option, so it won't throw for 400+ http odes

Reply:
It was my initial approach, but I think we want it to throw an error for anything which is not 404, like 401

Reply:
OK, is there any api description, which http codes are returned? I assume it will now always return 200 below, so the condition isn't needed.

Reply:
Yes, 200 if member, 404 if not:
https://developer.atlassian.com/cloud/bitbucket/rest/api-group-workspaces/#api-workspaces-workspace-members-member-get-responses

401 is also documented, I would assume 403 is also possible (e.g. misconfigured App password), and as a general rule, it is hard to know what else could go wrong, thus I did not want errors to be silently ignored

Please do not hesitate to let me know if you still think there is a better approach

Reply:
all 400+ code will throw, so i really expect to not have other success codes than `200`, to remove the  200 check below